README.md

```diff
@@ -1,23 +1,56 @@
-# Description
+Interchain query relayer implementation for [Neutron](https://github.com/neutron-org/neutron).
 
-Interchain query relayer implementation for Neutron
+More on relayer's configuration and inner can be read in [neutron-docs](https://github.com/neutron-org/neutron-docs/blob/main/docs/relaying/icq-relayer-guide.md) 
```

Comment:
yes, it's better to use the link to the github pages: https://neutron-org.github.io/neutron-docs/relaying/icq-relayer